keel-echo/src/test/kotlin/com/netflix/spinnaker/keel/echo/NotifierTests.kt

```diff
@@ -78,7 +76,7 @@ class NotifierTests : JUnit5Minutests {
 
       context("new notification") {
         before {
-          every { notificationRepository.addNotification(event.scope, event.ref, event.type)} returns true
+          every { notificationRepository.addNotification(event.scope, event.ref!!, event.type)} returns true
```

Comment:
I think you can revert these `!!` since ref is required in the `RepeatedNotification`